sources/web/datalab/polymer/components/datalab-files/datalab-files.html

```diff
@@ -174,10 +174,13 @@
         <iron-icon icon="add-box"></iron-icon>
         <span>Folder</span>
       </paper-button>
-      <paper-button class="toolbar-button">
+      <paper-button class="toolbar-button" on-click="_altUpload">
```

Comment:
Sure. The upload button is still visible though. This is just a trick to avoid having the user interact with the ugly `<input>` element.